lib/open_api_spex/schema_helper.ex

```diff
@@ -0,0 +1,10 @@
+defmodule OpenApiSpex.SchemaHelper do
+  def convert_type(v) when is_list(v), do: :array
```

Comment:
Not sure if this function needs to be in a separate module.

`typeof` or `term_type` might be a better name.

Reply:
👍